client/src/components/Forms.js

```diff
@@ -81,11 +78,11 @@ const LogInForm = (props) => {
 const StartForm = (props) => {
   return ( 
     <div className="card">
```

Comment:
Could this be a reactstrap <Card> or does it need to be HTML?

Reply:
Needs to be HTML.  Reactstrap is very tricky with cards.

Reply:
what does it do if you add className="card" to the <Card>? That could be a work around while keeping the reactstrap paradigm